src/docfx/lib/github/GitHubUserCache.cs

```diff
@@ -135,11 +137,20 @@ public Task SaveChanges()
 
             lock (_lock)
             {
-                var file = new GitHubUserCacheFile
+                var file = JsonUtility.Serialize(new GitHubUserCacheFile
                 {
                     Users = Users.ToArray(),
+                });
+                var tasks = new List<Task>
+                {
+                    Task.Run(() => ProcessUtility.WriteFile(_cachePath, file)),
```

Comment:
>Task.Run(() [](start = 20, length = 11)

don't use task.Run [https://stackoverflow.com/questions/11706788/should-methods-returning-taskt-always-start-the-returned-task/11707546#11707546](https://stackoverflow.com/questions/11706788/should-methods-returning-taskt-always-start-the-returned-task/11707546#11707546)